tests/modules/usercall.c

```diff
@@ -0,0 +1,170 @@
+#include "redismodule.h"
+#include <stdio.h>
+#include <strings.h>
```

Comment:
Not sure this include is needed, please check.

Reply:
removed both and the printf for debugging